src/more-infos/more-info-climate.html

```diff
@@ -269,13 +293,16 @@
         !stateObj.attributes.target_temp_high;
   }
 
-  computeHideTempRangeSlider(stateObj) {
-    return !stateObj.attributes.target_temp_low &&
-        !stateObj.attributes.target_temp_high;
+  computeHideTempRangeControl(stateObj) {
+    return (!stateObj.attributes.target_temp_low &&
+        !stateObj.attributes.target_temp_high) ||
+        (stateObj.attributes.operation_mode !== 'auto' && stateObj.attributes.temperature);
```

Comment:
I don't think we should special-class `auto` it doesn't necessary means enabling temperature range.

Reply:
I was running tests based on generic_thermostat and the "demo" platform with it's available 4 types of thermostats. 

There are thermostats with Auto that have low & high, and thermostats with Auto with only target temperature.

Reply:
The UI should represent the climate component generally:
The target-range feature is independent of the operation modes.
Also, instead of (or in addition to) checking for properties check  `supported_features` attribute.